src/api-sdk/github/utils.ts

```diff
@@ -0,0 +1,85 @@
+import { ApiResponse, ApiErrorResponse } from '../response';
+
+export const extractBaseShaHelper = (response: ApiResponse<any> | ApiErrorResponse) => {
+    if ('resource' in response && response.resource && 'object' in response.resource && 'sha' in response.resource.object) {
+        return response.resource.object.sha;
+    } else {
+        return response;
+    }
+};
+
+export const extractShaHelper = (response: ApiResponse<any> | ApiErrorResponse) => {
+    if ('resource' in response && response.resource && 'sha' in response.resource) {
+        return response.resource.sha;
+    } else {
+        return response;
+    }
+};
+
+export const getShaParams = (repoUrl: string) => {
+    const shaUrl = `${repoUrl}/git/refs/heads/main`;
```

Comment:
Create a default value for the main/master brunch and update `getShaParams` and `createPullRequestParams`. 

```suggestion
const defaultBranch = 'main';
...

export const getShaParams = (repoUrl: string, branchName: string = defaultBranch) => {
    const shaUrl = `${repoUrl}/git/refs/heads/${branchName}`;
    return shaUrl;
};

...


```